apps/web/app/lib/localization/translations.ts

```diff
@@ -68,48 +67,48 @@ const en = {
   'contact.form.submit': 'Send it',
   'contact.form.error':
     'An error occurred while sending your message. Please try again later.',
-  'contact.form.success.title': 'Thank you for your message!',
+  'contact.form.success.title': 'Thank you for contacting us!',
   'contact.form.success.description':
-    'Your message has been sent successfully!',
+    'Your message has been sent successfully! We will take notice of it and answer as soon as possible.',
   'give.title': 'Support the {{movement}}.',
   'give.movement': 'movement',
   'give.reason':
-    'Our call to mission, is to be the light of the world. Not a light hidden in the corner, covered by the humdrum of life, but in the open, where all can see. GYC Canada wants to magnify that light, to bring it to all of Canada.',
+  "Our call to mission, is to be the light of the world. Our Savior calls us to a worldwide mission of spreading the gospel of Christ's love in this world filled with darkness. This light should not be hidden in a corner, covered by the humdrum of life, but out in the open, where all can see and hear. GYC Canada wants to magnify that light, to bring it to all of Canada and the world.",
```

Comment:
I rephrased this a little because I though the hiding of the light in a corner sounded a little funny.